### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by jmeas** [jmeas/redux-resource#209]
I'd love to have a logo to represent this lib :v:

Some thoughts:

- a nod to the Redux logo could make sense, but isn't necessary
- the notion of a "resource" doesn't really have a set image that I'm aware of. Two ideas are basic shapes (sphere, pyramid, box), which could represent how resources are abstract things of different kinds. Or maybe a document, since resources on the web are often transferred within the body of an HTTP request.

shrug – I'm open to more ideas or possible logos :)

**Comment by sprjr** [comment]
The first thing I thought of turns out to be a nod to the redux logo. Theirs is a bit more 'reacty' but I was thinking the idea of just 3 dots/boxes/shapes arranged in a circle with some arrows on the lines indicating 'direction'. The thought there was just that our resources tend to make trips back and forth between machines while they're being CRUDed.

+shrug

**Comment by jmeas** [comment]
Still interested in this, but I'll be closing this issue out and labeling it revisit. Open to more ideas still though! :v:

**Comment by jmeas** [comment]
I made the logo this weekend! I thought I would share the process of creating it.

First, I thought about some features of the logo that I wanted:

1. it should reference connections/graphs/relationships/documents in some way
2. it should look good at small sizes
3. it would be nice if it worked in one color
4. a nod to the Redux logo would be nice, but is not necessary

---

I did some sketches:

![osyeplrxtlstbppe2fng q](https://user-images.githubusercontent.com/2322305/35490645-0081aeb8-0456-11e8-80e3-e591ecbfae08.jpg)

You'll see the React logo there, because it does a great job at capturing 1 and 2 above, and I wanted to keep that simplicity in mind while thinking about this lib's logo. There's also the OpenVis logo, which I think is a nice logo for representing graphs/relationships, but it doesn't really meet the first requirement. The hexagon there is a nod to GraphQL's logo, which I think does a great job the first 3 items on the list.

Anyway, at some point I felt pretty confident I would end up going with the dotted double-R one, which is why there are a lot of different variations of it.

![fullsizeoutput_c8e](https://user-images.githubusercontent.com/2322305/35490692-503943ee-0456-11e8-8c43-3b84411f70e4.jpeg)

At some point, I decided to try out a "connected" version of the trefoil shape from the Redux logo. I put the dots "randomly" on it at first:

![fullsizeoutput_c90](https://user-images.githubusercontent.com/2322305/35490722-7c0f5b34-0456-11e8-849e-d16e07a136da.jpeg)

before I decided on putting the dots where the lines intersect. And there's when we ended up with this logo. It actually went through a large number of variations of colors and line thickness.

Ultimately, I settled on what ya see on the website and the README:

![logo-with-type-new-thick](https://user-images.githubusercontent.com/2322305/35490762-cac68e6e-0456-11e8-9b49-a906ba830076.png)

And that's how we ended up with this logo. I hope ya like it :v:
